content/show/coder-radio/_index.md

```diff
@@ -19,11 +19,14 @@ podverse_podcast_id = "ZXd_1Ojd9"
 [links.twitter]
   url = "https://twitter.com/coderradioshow"
 [links.email]
-  url = "https://www.jupiterbroadcasting.com/contact/"  
+  url = "https://coder.show/contact"  
 [links.reddit]
   url = "https://www.reddit.com/r/coderradio"  
 [links.shownotes]
   url = "https://coder.show/"  
+[links.youtube]
+  url="https://www.youtube.com/watch?v=QQwdoS__pss&list=PLCB9545FD2C26E547"
```

Comment:
seems this link is formatted toward a specific video w playlist, not a playlist itself.... great upgrade idea though from just the standard JB YT link

get the playlist URL then we're 🎉 !

Reply:
I did not realize there was a separate list url. Thanks.